### PR TITLE
Postgres setup

### PR DESCRIPTION
**Pull request by o0Maikeru0o**
-Installed postrgresSQL with server running on port 5432
-Installed sequelize and connected it to postgres server, no username or password
-Replaced legacy database (MySql) schema with sequelize model
-Created CRUD functions getAllItems, getItemById, getItemByName, createItem, updateItem, deleteItem
-Updated readme to reflect changes

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -1,52 +1,115 @@
...
+
+const createItem = item => Item.findOrCreate({ where: { name: item.name } })
+  .spread((newItem, created) => newItem)
+  .catch(err => console.log(err));
+
+const updateItem = (id, params) => Item.Update({ params }, { where: { id } })
+  .then(updated => updated)
+  .catch(err => console.log(err));
+
+const deleteItem = id => Item.destroy({ where: id })
```
Comment (merlinv12) [inline database/index.js:58]: destroy!!!

File: database/index.js [hunk 2]
```diff
@@ -1,52 +1,115 @@
...
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  description: Sequelize.STRING,
+  fabric: Sequelize.STRING,
+  care: Sequelize.STRING,
+  features: Sequelize.STRING,
+  colors: Sequelize.STRING,
+  prize: Sequelize.STRING,
```
Comment (merlinv12) [inline database/index.js]: prize = price?

File: database/index.js [hunk 3]
```diff
@@ -1,52 +1,115 @@
...
+  })
+  .catch((err) => {
+    console.error('Unable to connect to the database:', err);
   });
-};
+
+const { Model } = Sequelize;
+
+class Item extends Model {}
+Item.init({
```
Comment (merlinv12) [inline database/index.js:17]: Looks like you are defining your schema here using Sequelize.  Make sure that you add parameters for VARCHAR.  I think by default VARCHAR is 255, which might not be long enough for description.

**Review by merlinv12 — APPROVED** [review]
Looks good to me.  Fix that typo in the Schema.

File: database/index.js (hunk 2 above)
Comment (o0Maikeru0o) [reply database/index.js]: Good catch thanks

File: database/index.js [hunk 4]
```diff
@@ -1,52 +1,115 @@
...
+  id: {
+    type: Sequelize.INTEGER,
+    primaryKey: true,
+    autoIncrement: true,
+  },
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  description: Sequelize.STRING,
```
Comment (o0Maikeru0o) [inline database/index.js]:
```suggestion
  description: Sequelize.STRING(500),
```

File: database/index.js (hunk 2 above)
Comment (o0Maikeru0o) [inline database/index.js]:
```suggestion
  price: Sequelize.STRING,
```

File: database/index.js (hunk 3 above)
Comment (o0Maikeru0o) [reply database/index.js:17]: Good call, I'll bump it up to 500

**Pull request merged**